### DISCUSSION TITLE
install failed

### DISCUSSION THREAD

**Discussion by jim123ture** [pezzolabs/pezzo#200] [Q&A]
opensearch-dashboards    | {"type":"log","@timestamp":"2023-09-16T16:23:38Z","tags":["error","opensearch","data"],"pid":1,"message":"

**Comment by arielweinberger** [comment]
Hi @jim123ture please report issues in the [issues section](https://github.com/pezzolabs/pezzo/issues), and make sure to provide detailed logs.
